### DISCUSSION TITLE
When will a release become available for install via a package manager?

### DISCUSSION THREAD

**Discussion by wsandy1** [cockpit-project/cockpit#17052] [Q&A] [answered]
### Explain what happens

Hi, sorry for the perhaps silly question. I'm waiting for something which I believe is in release 264. How long will it be until it is available for install with yum? Or am I missing something altogether? Will it be after a later major release?

### Server operating system

CentOS

**Comment by jelly** [comment]
Which CentOS version? 8 or 9?

**Reply by wsandy1** [reply]
Neither, Rocky Linux but basically CentOS 8 :)

**Comment by martinpitt** [comment, marked as answer]
I released 264 just a few hours ago. [CentOS 9 stream update](https://gitlab.com/redhat/centos-stream/rpms/cockpit/-/merge_requests/19) is in the works (currently waiting for CI). We haven't started the c8s update yet, but that's still the plan today.

**Reply by wsandy1** [reply]
Ah I see so you're expecting to have the 8 stream updated today?

**Reply by martinpitt** [reply]
I triggered the RHEL 8 build, and it just succeeded. Will take a few hours to beat it through gating. From then on it get auto-imported into C8S. So, "soon", but I don't have a precise ETA.

(Out of interest: Anything wrong with 263 that you are so eager for 264? :wink: )

**Reply by wsandy1** [reply]
Thanks 👍 I'll keep an eye :)

And not at all, I've just been waiting for the support of the XFF header so I can (hopefully) set up fail2ban for the cockpit login 😬
